CHANGES_NEXT_RELEASE

```diff
@@ -2,6 +2,7 @@
 - Add: new option to set the timeout for REST request: -reqTimeout (#2762)
 - Fix: modified the upper limit for CLI '-maxConnections', from 1020 to UNLIMITED
 - Fix: proper 500 Internal Server Error response with descriptive text when "sort fail" at DB occur (#2752)
+- Fix: avoid to present empty metrics for services (#2846)
```

Comment:
Given that the Metrics API has been developed as part as this sprint, this CNR line is not needed. I mean, the line:

> Add: metrics REST API (#2750)

already covers it.